package.json

```diff
@@ -32,7 +32,8 @@
     "@types/hast": "^2.3.4",
     "@types/recharts": "^1.8.23",
     "@types/react": "^17.0.59",
-    "@types/unist": "^2.0.6"
+    "@types/unist": "^2.0.6",
+    "graphql": "^16.2.0"
```

Comment:
Why do we need this? I'm probably missing something obvious

Reply:
GraphQL, like React, is very realms sensitive. If the backend encounters another GraphQL version, it fails. That failure is built deep into the toolchain, so it must encounter the same version everywhere. Without this, we'll get an older version from graphiql and api-docs and a newer version for GraphQL from the graphql-backend. Resolution ensures that only one version is used everywhere. I'm not sure why we're using an older version here even though upstream is on a newer version of GraphQL.

Reply:
That makes sense to me, I should have looked up that this was under the resolutions, had assumed this was a dependency